test/groovy/CloudFoundryDeployTest.groovy

```diff
@@ -175,21 +169,17 @@ class CloudFoundryDeployTest extends BasePiperTest {
                 manifest: 'test.yml'
             ]
         ])
-
-        assertEquals('s4sdk/docker-cf-cli', jedr.dockerParams.dockerImage)
-        assertEquals('/home/piper', jedr.dockerParams.dockerWorkspace)
-        assertEquals('200', jedr.dockerParams.dockerEnvVars.STATUS_CODE.toString())
-
-
-        assertTrue(jscr.shell[1].contains('cf login -u "test_cf" -p \'********\' -a https://api.cf.eu10.hana.ondemand.com -o "testOrg" -s "testSpace"'))
-        assertTrue(jscr.shell[1].contains('cf push "testAppName" -f "test.yml"'))
+        // asserts
```

Comment:
unrelated to scope of PR, only another way of asserting --> should be done in a dedicate PR.

Reply:
needed to change assertion because of `assertTrue(jscr.shell[0].contains...`